test/tools/solc_target.cpp

```diff
@@ -78,6 +117,12 @@ namespace
 extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
 {
     string input(reinterpret_cast<const char*>(data), size);
-    testCompiler(input, true);
+
+    if (size % 3 == 0)
```

Comment:
Is it possible to just have different runs for these different components? We plan to add another one, and just making this based on the size sounds a little weird. If there is no other way, we could use a single byte at the beginning (which is then chopped off) to distinguish the different components to test.

Reply:
I see what you mean. We could use the first byte exclusively for choosing which test to run. The (minor) downside of this approach is that the fuzzer payload is not pure solidity code (second byte onwards is). Triage scripts may need to be updated to take this into account.

Alternatively, one could create separate test harnesses (one harness per component) for each of the said components i.e., compiler with/without optimization, and constant optimizer.